Solutions/SOC-Process-Framework/Playbooks/readme.md

```diff
@@ -22,4 +22,4 @@ After deploying the playbook, you must authorize the connections leveraged.
 2. Under "Development Tools" (located on the left), click "API Connections".
 3. Ensure each connection has been authorized.
 
```

Comment:
We think that for playbook to add task to incident , it should have "sentinel responder role" assigned .

Kindly add the steps here on how to assign the sentinel responder role .

Ref : https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-Enrichment/readme.md

Reply:
Hi, done